db/post_migrate/20230119091939_drop_orphaned_reviewable_flagged_posts.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class DropOrphanedReviewableFlaggedPosts < ActiveRecord::Migration[7.0]
+  def up
+    DB.exec(<<~SQL)
+      DELETE FROM reviewables
+      WHERE reviewables.type = 'ReviewableFlaggedPost'
+        AND reviewables.status = 0
+        AND reviewables.target_type = 'Post'
+        AND (NOT EXISTS(SELECT * FROM posts WHERE posts.id = reviewables.target_id))
```

Comment:
I've always used `EXISTS(SELECT 1` as I was under the impression it was less work / faster since we only care about the existence and don't need to return all the columns. Though I wonder if the SQL engine is clever enough to automagically optimise this check? 🤔 

```suggestion
        AND NOT EXISTS(SELECT 1 FROM posts WHERE posts.id = reviewables.target_id)
```

Reply:
Actually, that’s a good question, I’ll check if there’s a big difference between the two versions :+1:

Reply:
I applied your suggestion but when running `EXPLAIN` on both versions, the output stays the same :sweat_smile:

Reply:
I guess the optimiser is doing its job?